package.json

```diff
@@ -12,7 +12,7 @@
 		"url": "https://github.com/aws/amazon-states-language-service"
 	},
 	"license": "MIT",
-	"version": "1.1.4",
+	"version": "1.1.5",
```

Comment:
since you are adding additional options I think it should be minor version change and not a patch change.